perllib/FixMyStreet/Map/Buckinghamshire.pm

```diff
@@ -2,19 +2,88 @@
 # More JavaScript, for street assets
 
 package FixMyStreet::Map::Buckinghamshire;
-use base 'FixMyStreet::Map::OSM';
+use base 'FixMyStreet::Map::WMTSBase';
 
 use strict;
 
+sub zoom_parameters {
+    my $self = shift;
+    my $params = {
+        zoom_levels    => scalar $self->scales,
+        default_zoom   => 5,
+        min_zoom_level => 0,
+        id_offset      => 0,
+    };
+    return $params;
+}
+
+sub tile_parameters {
```

Comment:
Given how most of this fn is shared with Bristol (plus the reproject, zoom_parameters functions), worth having a parent class used by both?